doc/source/placement-group.rst

```diff
@@ -115,6 +115,18 @@ Let's see an example of using placement group. Note that this example is done wi
 
 Let's create a placement group. Recall that each bundle is a collection of resources, and tasks or actors can be scheduled on each bundle.
 
+.. note::
+
+  When specifying bundles,
+
+  - "CPU": is corresponding to num_cpus
+  - "GPU": is corresponding to num_gpus
+  - "MEM": is corresponding to memory
+  - Other resources are corresponding to custom resources.
+
+  Once placement group reserves resources, original resources are unavilable until placement group is removed. For example,
+  if you create {"CPU": 2} bundle on node A of 2 `num_cpus`, node A will not be able to non-placement group tasks and actors that require 2 `num_cpus`.
```

Comment:
```suggestion
  Once the placement group reserves resources, original resources are unavailable until the placement group is removed. For example,
  if you create a `{"CPU": 2}` bundle on node A of 2 `num_cpus`, node A will not be able to place non-placement group tasks and actors that require 2 `num_cpus`.
```

Reply:
Actually, I don't understand this paragraph...

Reply:
I just wanted to explain the concept of pre-reserve to people who are new. What about this one? 

```
# Two "CPU"s are available.
ray.init(num_cpus=2)

# Create placement group.
pg = placement_group([{"CPU": 2}])
ray.get(pg.ready())

# Now, 2 CPUs are not available anymore because they are pre-reserved by the placement group.

@ray.remote(num_cpus=2)
def f():
    return True

# Will not be scheduled.
f.remote()

# Will be scheduled.
f.options(placement_group=pg).remote()
```

Reply:
Ah looks very good :)